### DISCUSSION TITLE
What's the primary key / unique key?

### DISCUSSION THREAD

**Discussion by csirmazbendeguz** [nager/Nager.Date#717] [Q&A] [answered]
Hi! 👋

Thanks for your work on this library! It's an awesome project.
I've been doing some testing today and I have a question.

I'm planning to store the public holidays in a database.
What fields do you recommend I use for the primary key or unique key?

I was thinking of placing a unique constraint on all fields, but perhaps that's an overkill as some fields (e.g. `launchYear`, `localName`, `global`, `fixed`) don't look like they _identify_ a public holiday.

**Comment by tinohager** [comment]
I am considering introducing a unique ID for each public holiday. But I haven't gotten around to implementing it yet.

- https://github.com/nager/Nager.Date/issues/201

**Reply by csirmazbendeguz** [reply]
I see, thanks for the quick reply!

**Comment by tinohager** [comment, marked as answer]
- https://github.com/nager/Nager.Date/pull/864
